cookbooks/aws-parallelcluster-slurm/templates/default/slurm/resume_program.erb

```diff
@@ -1,2 +1,13 @@
 #!/bin/bash
-sudo -u <%= node['cluster']['cluster_admin_user'] %> <%= node_virtualenv_path %>/bin/slurm_resume "$@"
+# ResumeProgram should read SLURM_RESUME_FILE within ten seconds of starting to guarantee that it still exists.
+# ref https://slurm.schedmd.com/power_save.html#tolerance
+
+trap "rm -f ${SLURM_RESUME_FILE_TMP}" EXIT
+
+SLURM_RESUME_FILE_TMP=$(mktemp)
```

Comment:
What about providing a file name pattern, such as `slurm-resume-file-XXXXXXX`, to make it easier to discover this type of files in temp dir?

Reply:
the file will be deleted at the end of the resume execution, so I thought it's not useful to have a different name